zerver/webhooks/gocd/doc.md

```diff
@@ -0,0 +1,21 @@
+Zulip supports integration with GoCD and can notify you of
+your build statuses.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Add the following to your `Config.XML` file:
+
+    ```
+    <pipeline name="mypipeline">
+        <trackingtool link="http://mingle.thoughtworks.com/go/${ID}" regex="##(\d+)"/>
+        ...
+    </pipeline>
+    ```
+
+    For further information, see [GoCD's documentation](https://docs.gocd.org/current/integration/)
```

Comment:
Is `Config.XML` a part of the repository for which you'd like to receive build notifications? If so, we should probably add `Push this change to your repository.` at the end.

Also could you please add a period at the end of sentence that begins with `For further information...`. :)

Reply:
Also, where does the webhook URL go?

Reply:
@eeshangarg updated the changes. And squashed into one commit.